third_party/move/move-prover/boogie-backend/src/spec_translator.rs

```diff
@@ -973,14 +973,28 @@ impl<'env> SpecTranslator<'env> {
                 // Skip freeze operation
                 self.translate_exp(&args[0])
             },
+            Operation::Vector if args.is_empty() => {
+                self.translate_primitive_inst_call(node_id, "$EmptyVec", args)
+            },
+            Operation::Vector if args.len() == 1 => self.translate_primitive_call("MakeVec1", args),
```

Comment:
We have up to `MakeVec4` and they are more efficient, so lets use them if we can.